### PR TITLE
Update Nexus package for Sonatype ownership.

### PR DESCRIPTION
**Pull request by whyjustin**
Update the package and configuration for Sonatype ownership.

### CODE REVIEW

File: repo/packages/N/nexus/1/resource.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "images": {
+    "icon-small": "http://www.sharelytics.de/assets/img/nexus/nexus_48.png",
+    "icon-medium": "http://www.sharelytics.de/assets/img/nexus/nexus_96.png",
+    "icon-large": "http://www.sharelytics.de/assets/img/nexus/nexus_256.png"
```
Comment (ryadav88) [inline repo/packages/N/nexus/1/resource.json]: please use assets backed by a cdn. maybe upload these images to your s3 bucket and use the public link from there?

File: repo/packages/N/nexus/1/package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+  "packagingVersion": "3.0",
+  "name": "nexus",
+  "version": "3.1.0",
+  "minDcosReleaseVersion" : "1.8",
+  "scm": "https://github.com/sonatype/docker-nexus3",
+  "maintainer": "cloud-ops@sonatype.com",
+  "website": "https://www.sonatype.com/",
+  "framework": false,
+  "description": "Nexus Repository Manager is the world's only repository manager with FREE support for popular formats. It supports Bower, Docker, Maven 2, npm, NuGet, PyPI, and Raw site repositories.",
```
Comment (ryadav88) [inline repo/packages/N/nexus/1/package.json]: add this at the end of `description`: `Installation documentation can be found at https://github.com/dcos/examples/tree/master/1.8/nexus`

Comment (ryadav88) [inline repo/packages/N/nexus/1/package.json]: Also, can you review installation documentation at https://github.com/dcos/examples/tree/master/1.8/nexus. please update the installation steps there if anything has changed from the previous version.

Comment (whyjustin) [inline repo/packages/N/nexus/1/package.json]: > add this at the end of description: Installation documentation can be found at https://github.com/dcos/examples/tree/master/1.8/nexus

Added f96d20f

> Also, can you review installation documentation at https://github.com/dcos/examples/tree/master/1.8/nexus. please update the installation steps there if anything has changed from the previous version.

PR out in the Example repository https://github.com/dcos/examples/pull/37

File: repo/packages/N/nexus/1/resource.json (hunk 1 above)
Comment (whyjustin) [inline repo/packages/N/nexus/1/resource.json]: > please use assets backed by a cdn. maybe upload these images to your s3 bucket and use the public link from there?

We are working internally to get these available on a CDN. Will update when ready.

Comment (whyjustin) [inline repo/packages/N/nexus/1/resource.json]: > please use assets backed by a cdn

Updated to use Sonatype CDN 65ba4b3

**Pull request merged**